signer.sh

```diff
@@ -26,7 +26,8 @@ CONTAINER_NAME="aleph-signer"
 SIGNER_VERSION="fca5dd5"
 NODE_KEY_PATH="/data/p2p_secret"
 SIGNER_IMAGE=public.ecr.aws/p6e8q1z1/peer-verifier:${SIGNER_VERSION}
-HOST_SECRET_PATH="$(pwd)/p2p_secret"
+ALEPH_DIRECTORY="${HOME}/.alephzero"
```

Comment:
that seems to be the default, but what if it was overridden (as per -d | --data_dir argument to th enode runner script)?

Reply:
Well, then it would make this whole operation a no-op, which is not the end of the world (although doesn't make much sense)